src/base/Constants.hh

```diff
@@ -23,18 +23,19 @@ namespace constants
  * System of Units, 9th ed., for definition of constants and how they relate to
  * the different units.
  *
- * Celeritas            | CLHEP                 | Notes
- * -------------------- | --------------------- | ------------
- * a0_bohr              | Bohr_radius           | Bohr radius
- * alpha_fine_structure | fine_structure_const  | |
- * atomic_mass          | amu                   | Not the same as 1/avogadro
- * eps_electric         | epsilon0              | Vacuum permittivity
- * h_planck             | h_Planck              | |
- * k_boltzmann          | k_Boltzmann           | |
- * mu_magnetic          | mu0                   | Vacuum permeability
- * na_avogadro          | Avogadro              | [1/mol]
- * re_electron          | classic_electr_radius | Classical electron radius
- * kcd_luminous         | [none]                | Lumens per Watt
+ * Celeritas            | CLHEP                   | Notes
+ * -------------------- | ---------------------   | ------------
+ * a0_bohr              | Bohr_radius             | Bohr radius
+ * alpha_fine_structure | fine_structure_const    | |
+ * atomic_mass          | amu                     | Not the same as 1/avogadro
+ * eps_electric         | epsilon0                | Vacuum permittivity
+ * h_planck             | h_Planck                | |
+ * k_boltzmann          | k_Boltzmann             | |
+ * mu_magnetic          | mu0                     | Vacuum permeability
+ * na_avogadro          | Avogadro                | [1/mol]
+ * re_electron          | classic_electr_radius   | Classical electron radius
+ * kcd_luminous         | [none]                  | Lumens per Watt
+ * lambda_compton       | electron_Compton_length | Reduced Compton wavelength
```

Comment:
Since the Compton warelength is particle-dependent, I would still put "electron" to be specific.

Reply:
Good point, will suffix with an `_e`. Interestingly the CODATA "fundamental physical constants" listing doesn't qualify the name with "electron" -- it just reads "reduced Compton wavelength $\hbar / m_e c = α a_0$".